src/site/stages/build/drupal/graphql/healthCareRegionPage.graphql.js

```diff
@@ -8,7 +8,12 @@ const healthCareLocalFacilities = require('./facilities-fragments/healthCareLoca
 const healthCareRegionHealthServices = require('./facilities-fragments/healthCareRegionHealthServices.node.graphql');
 const healthCareRegionNewsStories = require('./facilities-fragments/healthCareRegionNewsStories.node.graphql');
 const healthCareRegionEvents = require('./facilities-fragments/healthCareRegionEvents.node.graphql');
-const healthCareStaffBios = require('./facilities-fragments/healthCareRegionStaffBios.node.graphql');
+// const healthCareStaffBios = require('./facilities-fragments/healthCareRegionStaffBios.node.graphql');
+
+const healthCareStaffBios = requireWithFeatureFlag(
+  __dirname,
```

Comment:
This is required because otherwise you'll require a file relative to the `featureFlags.js` file.

Reply:
Yeah, I saw the stack trace approach and felt like that was a bit too strange. This is an extra argument, but the code is at least somewhat clear.